wled00/FX_fcn.cpp

```diff
@@ -1115,6 +1126,7 @@ void WS2812FX::service() {
 
       seg.next_time = nowUp + delay;
     }
+    seg.resetIfRequired(); // another reset chance, mainly to apply new segment bounds if queued
```

Comment:
unnecessary

Reply:
Necessary, but only for applying queued changes immediately after the FX function is done. Will be changed, as I will also move applying queued changes out of `resetIfRequired()`